docs/user/FlowVariables.md

```diff
@@ -0,0 +1,268 @@
+| `TAPCELL_TCL`         | Path to Endcap and Welltie cells file.                                                                                                                                           |
+| `MACRO_PLACE_HALO`    | horizontal/vertical halo around macros (microns).                                                                                                                                |
+| `MACRO_PLACE_CHANNEL` | horizontal/vertical channel width between macros (microns).                                                                                                                      |
+| `PDN_CFG`             | File path which has a set of power grid policies used by `pdn` to be applied to the design, such as layers to use, stripe width and spacing to generate the actual metal straps. |
+| `MAKE_TRACKS`         | Tcl file that defines add routing tracks to a floorplan.                                                                                                                         |
+| `IO_PIN_MARGIN`       | Maximum number of I/O pins placed outside floorplan boundary. Integer value.                                                                                                     |
+| `IO_PLACER_H`         | The metal layer on which to place the I/O pins horizontally (top and bottom of the die).                                                                                         |
+| `IO_PLACER_V`         | The metal layer on which to place the I/O pins vertically (sides of the die).                                                                                                    |
+
+
+### Placement
+
+
+| Variable                             | Description                                                                                                                                   |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| `CELL_PAD_IN_SITES_GLOBAL_PLACEMENT` | Cell padding on both sides in site widths to ease routability during global placement. Default 2 sites.                                       |
+| `CELL_PAD_IN_SITES_DETAIL_PLACEMENT` | Cell padding on both sides in site widths to ease routability in detail placement. Default 1 sites.                                           |
+| `MAX_WIRE_LENGTH`                    | Specifies the maximum wire length used by the resizer to reduce RC delay by inserting buffers. Integer value.                                 |
+| `WIRE_RC_LAYER`                      | Metal Layer to use for parasitics estimations.                                                                                                |
+| `CELL_PAD_IN_SITES`                  | Cell padding in SITE widths to ease rout-ability. Integer value.                                                                              |

```

Comment:
This was replaced by CELL_PAD_IN_SITES_GLOBAL_PLACEMENT & CELL_PAD_IN_SITES_DETAIL_PLACEMENT